components/raftstore-v2/src/raft/peer.rs

```diff
@@ -24,6 +27,8 @@ pub struct Peer<EK: KvEngine, ER: RaftEngine> {
     /// peer list, for example, an isolated peer may need to send/receive
     /// messages with unknown peers after recovery.
     peer_cache: Vec<metapb::Peer>,
+    raw_write_encoder: Option<RawWriteEncoder>,
+    proposals: ProposalQueue<Vec<CmdResChannel>>,
     pub(crate) async_writer: AsyncWriter<EK, ER>,
```

Comment:
how about moving public fields together

Reply:
They are group by functionality instead of accessibility.

Reply:
seems they aren't group now